src/alfasim_sdk/models.py

```diff
@@ -133,12 +133,12 @@ def data_model(*, caption: str, icon: Optional[str] = None) -> Callable:
 
         @data_model(icon='', caption='My Plugin')
         class MyModel:
-	        distance = Quantity(value=1, unit='m', caption='Distance')
+                distance = Quantity(value=1, unit='m', caption='Distance')
 
 
         @alfasim_sdk.hookimpl
         def alfasim_get_data_model_type():
-	        return [MyModel]
+                return [MyModel]
```

Comment:
Not a problem.
But "a tab" has been expanded into "4 spaces" maybe the indentation is not as pleasant as expected.
I just mention it because I remember that someone noticed a similar effect in a doc string.